test/acceptance/pages.py

```diff
@@ -553,6 +553,27 @@ def grade_entry(self, question, column):
 
         return source.text.strip(), value.text.strip()
 
+    def get_individual_entry(self, question, row):
```

Comment:
Can you just return this information in the existing grade_entry method? I'd also like the value to be returned in addition to the peer "name".